### DISCUSSION TITLE
Skip file with error / whitelist filetype to open

### DISCUSSION THREAD

**Discussion by vjadid** [sumatrapdfreader/sumatrapdf#2884] [Ideas]
Hi,

I have a folder containing `pdf`, `zip`, and `jpg `files. I open a single `pdf `file in the folder then navigate to the next file using keyboard shortcut for command ID `CmdOpenNextFileInFolder`.
I face an issue where when the next file in folder is a unsupported file (like `zip`), the program displays error but doesn't let me move on to the next file (after `zip`). I can only move to the previous file using `CmdOpenPrevFileInFolder`.

Is there a way to let sumatra pdf know to skip files on error, or to whitelist file types to iterate in a folder (so that `CmdOpenNextFileInFolder `will only consider specific filetype, say `pdf`, and ignore the rest) ?

**Comment by GitHubRulesOK** [comment]
hmm
 has been a problem for sometime but should have been addressed in https://github.com/sumatrapdfreader/sumatrapdf/issues/1449 perhaps check which version you are running.

**Reply by vjadid** [reply]
I'm running version v3.4.6 64-bit on Windows 10 21H2.

Ok seems its fixed in [#1449](https://github.com/sumatrapdfreader/sumatrapdf/issues/1449) but not in the [3.4.6 release](https://github.com/sumatrapdfreader/sumatrapdf/blob/3.4.6rel/src/SumatraPDF.cpp)  , so hopefully later version get it fixed.
